DecipherNow/gm-fabric-dashboard
# 1025 fix long route name not elliipsing in chrome

Resolves #1025

## Proposed Changes

  - Route column displays span svg icon, route name and vizbar that takes up the width of column.  I wrapped icon and name inside a flex container, to limit the width of name.
  
## How to Test

  - navigate to JVM view and confirm long route name is truncating correctly in chrome
  - checked all supporting browsers
